vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/StaffSearchSpace.java

```diff
@@ -0,0 +1,16 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import java.util.List;
+
+public class StaffSearchSpace {
```

Comment:
are the staff serach classes used anywhere?

Reply:
No, This class have not been used anywhere, There is a similar class present naming StaffSearchSpaceResult, which has same body as StaffSearchSpace , so that is being used every where.